backends/mongodb/README.md

```diff
@@ -0,0 +1,31 @@
+# MongoDB backend
+
+## Prerequisites
+
+To use MongoDB as a backend `libmongoc` 1.7.0 or higher should be [installed](http://mongoc.org/libmongoc/current/installing.html) first. Next, Netdata should be re-installed from the source. The installer will detect that the required libraries are now available.
+
+## Configuration
+
+To enable data sending to the mongodb backend set the following options in `netdata.conf`:
```

Comment:
I think you should capitalize MongoDB here, as you're not referencing a process/user/etc.

```suggestion
To enable data sending to the MongoDB backend set the following options in `netdata.conf`:
```